packages/page-coretime/src/Sale/SaleDetailsView.tsx

```diff
@@ -0,0 +1,158 @@
+// Copyright 2017-2025 @polkadot/app-coretime authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ChainName, SaleParameters } from '../types.js';
+
+import React, { useCallback } from 'react';
+
+import { Button, styled } from '@polkadot/react-components';
+
+import { PhaseName } from '../constants.js';
+import { useTranslation } from '../translate.js';
+import PhaseTable from './PhaseTable.js';
+import { SubScanButton } from './SubScanButton.js';
+
+const ResponsiveContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  gap: 10rem;
+  margin-top: 2rem;
+
+  @media (max-width: 1000px) {
+    flex-direction: column;
+  }
+`;
+
+const Title = styled.h3`
+  font-weight: bold;
+  margin-bottom: 1rem;
+`;
+
+const LinkWithLogo = ({ alt, href, logo }: { href: string, logo: string, alt: string }) => {
+  return (
+    <a
+      href={href}
+      rel='noopener noreferrer'
+      target='_blank'
+    >
+      <img
+        alt={alt}
+        height={25}
+        src={logo}
+      />
+    </a>
+  );
+};
+
+const providers = {
+  lastic: {
+    alt: 'Lastic',
+    href: (chainName: string) => `https://www.lastic.xyz/${chainName}/bulkcore1`,
+    logo: 'https://www.lastic.xyz/_next/image?url=%2Fassets%2FImages%2FLogos%2Flastic-logo.png&w=384&q=100'
+  },
+  regionx: {
+    alt: 'RegionX',
+    href: (chainName: string) => `https://app.regionx.tech/?network=${chainName}`,
+    logo: 'https://app.regionx.tech/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Flogo.8f0fd171.png&w=3840&q=75'
+  },
+  subscan: {
+    alt: 'Subscan',
+    href: (chainName: string) => `https://coretime-${chainName}.subscan.io/coretime_dashboard`,
+    logo: 'https://www.subscan.io/_next/image?url=%2Fwebsite%2Flogo-light.png&w=256&q=75'
+  }
+};
+
+const phases = {
+  [PhaseName.Renewals]: {
+    description: 'In this phase, core owners can renew existing cores at a fixed price to ensure continued operation in the next region. No new core purchases are permitted.',
+    name: 'Interlude/Renewals phase'
+  },
+  [PhaseName.PriceDiscovery]: {
+    description: 'The period during which cores are available for both purchase and renewal. The price is linearly declining price.',
```

Comment:
IMO `The price is linearly declining price.` sounds a bit off - maybe something like: `The price decreases linearly over time.`

Reply:
🤦🏽‍♀️  totally. The price is a price you know 😅  😆